docs/plans/2026-06-10-v9.3-implementation-plan.md

```
@@ -0,0 +1,1186 @@
+    Intact,   // node with same name found, normalized body hash matches
+    Modified, // node found, hash differs — "re-verify before relying"
+    Broken,   // node gone (or file gone/unparseable)
+}
+```
+
+Content-addressed lookup by `name` within the file means relocation/reformat within a
+file is automatically `Intact`. Cross-file moves read as `Broken` — accepted for v9.3.
+
+### Tier-0 identity block (SessionStart stdout, ≤200 tokens)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)